instance/static/html/instance/index.html

```diff
@@ -0,0 +1,73 @@
+<top-bar>
+  <ul class="title-area">
+    <li class="name">
+      <h1><a href="#">OpenCraft - Instances Manager</a></h1>
+    </li>
+  </ul>
+
+  <section class="top-bar-section">
+    <!-- Right Nav Section -->
+    <ul class="right">
+      <li class="active"><a href="/admin/logout/?next=/">Logout</a></li>
+    </ul>
+  </section>
+</top-bar>
+
+<div class="row">
+  <!-- Instances List -->
+  <div class="large-3 columns">
+    <ul class="side-nav" ng-if="instanceList">
+      <li ng-repeat="instance in instanceList"
+          ng-class="{active: instance == selected.instance}">
+        <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+      </li>
+      <li class="divider"></li>
+    </ul>
+
+    <div class="row" ng-if="!instanceList">
+      <p>...</p>
+    </div>
+  </div>
+
+  <!-- Selected instance details -->
+  <div class="large-9 columns" ng-if="selected.instance">
+    <div class="row">
+      <h3>
+        {{ selected.instance.name }}
+        <span ng-repeat="server in selected.instance.active_server_set">
+          <span class="status radius label"
+                ng-class="{warning: server.status != 'ready', success: server.status == 'ready'}">
+            {{ server.status }}
+          </span>
+        </span>
+      </h3>
+      <div class="panel">
+        <div class="row">
+          <div class="large-10 columns">
+            <ul class="instance-description">
+              <li>Instance URL: <a href="{{ selected.instance.url }}" target="_new">{{ selected.instance.url }}</a></li>
+              <li><a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> | {{ selected.instance.branch_name }}</li>
+            </ul>
```

Comment:
From @e-kolpakov : 

> Show a corresponding PR link somewhere (just for convenience)

Agreed, it would be nice. The issue with adding it currently is that the models don't have any notion of PRs themselves - only the functions that create or update the instance objects know about it, so it's not available to the UI. I'll have another look at this when I work on the review comments, to see if there is a way to add this that isn't too involved.